tests/utils.go

```diff
@@ -645,6 +646,7 @@ func CalculateNamespaces() {
 func SynchronizedBeforeTestSetup() []byte {
 	var err error
 	Config, err = loadConfig()
+	Arch = getArch()
```

Comment:
Can you plese move it to a regular `BeforeTestSetup`? The response of `getArch` is otherwise only visible on the one ginkgo node which executed this synchronized function.

Reply:
do you mean function `BeforeTestSuitSetup` in utils.go?

Reply:
Yes.